src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -62,68 +66,71 @@
+                         {:toValue 1
+                          :duration 150
+                          :easing (.-ease (animation/easing))
+                          :useNativeDriver true})
+       (animation/timing neg-anim-y
+                         {:toValue (if platform/desktop? connectivity-bar-height 0)
+                          :duration 150
+                          :easing (.-ease (animation/easing))
+                          :useNativeDriver true})])
+     ;; second param of start() - a callback that fires when animation stops
+     #(reset! to-hide? true))))
 
 (defn connectivity-status
-  [{:keys [connected?]} anim-translate-y]
-  (let [anim-translate-y (or anim-translate-y (animation/create-value 0))
+  [{:keys [connected?]} neg-anim-translate-y]
+  (let [neg-anim-translate-y (or neg-anim-translate-y (animation/create-value 0))
         anim-opacity     (animation/create-value 0)]
     (manage-visibility connected?
-                       anim-opacity anim-translate-y)
+                       anim-opacity neg-anim-translate-y)
     (reagent/create-class
      {:component-did-update
       (fn [comp]
         (manage-visibility (:connected? (reagent/props comp))
-                           anim-opacity anim-translate-y))
+                           anim-opacity neg-anim-translate-y))
       :reagent-render
-      (fn [{:keys [view-id message on-press-fn
-                   connected? connecting?] :as opts}]
+      (fn [{:keys [view-id message on-press-fn connected? connecting?] :as opts}]
         [react/animated-view {:style               (styles/text-wrapper
                                                     (assoc opts
                                                            :height (if platform/desktop?
-                                                                     anim-translate-y
-                                                                     35)
+                                                                     neg-anim-translate-y
+                                                                     connectivity-bar-height)
                                                            :background-color (if connected?
                                                                                colors/green
                                                                                colors/gray)
+                                                           ;;TODO how does this affect desktop?
+                                                           :transform      neg-anim-translate-y
                                                            :opacity anim-opacity
                                                            :modal? (= view-id :chat-modal)))
                               :accessibility-label :connection-status-text}
          (when connecting?
-           [react/activity-indicator {:animated     true
+           [react/activity-indicator {:animating    true
```

Comment:
could you please elaborate? does this mean it was not animated before?

Reply:
it can be removed because default is true

https://facebook.github.io/react-native/docs/0.59/activityindicator

Reply:
cool.. going to remove it

@flexsurfer turns out correct property for `activityIndicator` is `:animating` not `animated`. Weird that it is inconsistent in react-native with some other component in which opposite is true.

Anyway, I was dealing with the problem of `connecting/fetching` indicator and couldn't figure out where in code dealt with that. I was confused about what it would be called and poked around/searhed-docs thinking maybe it was called `activity-indicator` :)